enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -103,29 +109,45 @@
 (defn dump
   "Serialized metabase instance into directory `path`."
   ([path user]
-   (dump path user :active))
-  ([path user state]
+   (dump path user :active {}))
+  ([path user opts]
+   (dump path user :active opts))
+  ([path user state opts]
    (mdb/setup-db!)
+   (log/infof "BEGIN DUMP to %s via user %s" path user)
    (let [users       (if user
                        (let [user (db/select-one User
                                     :email        user
                                     :is_superuser true)]
                          (assert user (trs "{0} is not a valid user" user))
                          [user])
                        [])
-         tables (Table)
-         collections (Collection)] ;(select-collections users state)]
+         databases   (if (contains? opts :only-db-ids)
+                       (db/select Database :id [:in (:only-db-ids opts)])
+                       (Database))
```

Comment:
It it worth ordering these by `:id` or something, just so serializations are repeatable? Otherwise the order stuff comes back in is sort of up to the database

Reply:
Changed to order them all by `:id`.